<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLayerDialog</class>
 <widget class="QDialog" name="AddLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Layer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="horizontalSpacing">
      <number>18</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="3" column="2">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Settings:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTreeWidget" name="attributeList">
       <attribute name="headerDefaultSectionSize">
        <number>120</number>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose View:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Choose Attribute:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTreeWidget" name="viewList">
       <attribute name="headerDefaultSectionSize">
        <number>120</number>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QFrame" name="interpreteGroup">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Interprete as:</string>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QCheckBox" name="colorCheckBox">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Percentage Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="heightSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="heightCheckBox">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QComboBox" name="colorRamp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Black to Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White to Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Black to Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White to Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Black to Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White to Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Black to Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White to Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Black to Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White to Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White to Black</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rainbow</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="overdraw">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Is overdraw Layer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddLayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddLayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>heightSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>colorRamp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
